x/tokenregistry/keeper/keeper.go

```diff
@@ -95,24 +95,24 @@ func (k keeper) RemoveToken(ctx sdk.Context, denom string) {
 			updated = append(updated, t)
 		}
 	}
-	k.SetDenomWhitelist(ctx, types.Registry{
+	k.SetRegistry(ctx, &types.Registry{
 		Entries: updated,
 	})
 }
 
-func (k keeper) SetDenomWhitelist(ctx sdk.Context, wl types.Registry) {
+func (k keeper) SetRegistry(ctx sdk.Context, wl *types.Registry) {
 	store := ctx.KVStore(k.storeKey)
-	bz := k.cdc.MustMarshalBinaryBare(&wl)
-	store.Set(types.WhitelistStorePrefix, bz)
+	bz := k.cdc.MustMarshalBinaryBare(wl)
+	store.Set(types.RegistryPrefix, bz)
 }
 
-func (k keeper) GetRegistry(ctx sdk.Context) types.Registry {
-	var whitelist types.Registry
+func (k keeper) GetRegistry(ctx sdk.Context) *types.Registry {
```

Comment:
Return a struct here so that there's an empty value instead of nil. If a nil value can be returned we should return an error to check as well. Returning an error is not usually necessary in most keeper functions.

Reply:
Thanks. Done!